x/mint/types/keys.go

```diff
@@ -25,3 +31,13 @@ const (
 func GetAllowedAddressStoreKey(address string) []byte {
 	return append(AllowedAddressKey, []byte(address)...)
 }
+
+// GetDelegationKey creates the key for delegator bond with validator
+// VALUE: staking/Delegation
+func GetDelegationKey(delAddr sdk.AccAddress, valAddr sdk.ValAddress) []byte {
```

Comment:
Why is this in x/mint/types?

Reply:
good point. forgot to remove debug functions that i created to configure modules communication.  
will remove it.

Reply:
removed